tests/30rooms/13anonymousaccess.pl

```diff
@@ -79,6 +79,20 @@
       })->main::expect_http_403;
    };
 
+sub await_event_not_presence_for
+{
+   my ( $user, $room_id, $ignored_users ) = @_;
+   await_event_for( $user, sub {
```

Comment:
Which then makes this

    await_event_for( $user, room_id => $room_id, sub { ...